Cargo.toml

```diff
@@ -19,6 +19,9 @@ license = "Apache-2.0"
 xilem_core = { version = "0.1.0", path = "crates/xilem_core" }
 kurbo = "0.11.0"
 
+[workspace.lints]
+clippy.semicolon_if_nothing_returned = "warn"
```

Comment:
I would expect this to use:
```toml
[workspace.lints.clippy]
semicolon_if_nothing_returned = "warn"
```
As this is how it is always documented. Is there a specific reason we chose not to follow this example here?

Reply:
From my review perspective I figured that we can change it later when we have more lints, as right now it doesn't make much of a difference.

Reply:
I've seen it used both ways in the wild. It isn't just clippy that can be controlled here, but rustc warnings as well. I don't care though. I'd written it the way first and changed it before submitting.

Reply:
I think following the documentation makes sense here, but it's not that important either way